devito/finite_differences/differentiable.py

```diff
@@ -304,8 +325,46 @@ class Mul(DifferentiableOp, sympy.Mul):
     __sympy_class__ = sympy.Mul
     __new__ = DifferentiableOp.__new__
 
+    @property
+    def _gather_for_diff(self):
```

Comment:
I wonder whether this method should appear in the superclass too, clearly returning something dummy, or the object ``self``` itself?

Reply:
done